funnel/views/proposal.py

```diff
@@ -42,7 +42,7 @@
     'speaker',
     'email',
     'slides',
-    'preview_video',
+    'video_url',
```

Comment:
`video` right?

Reply:
This is used when generating CSV. Do we want to put a dict in there? I thought video_url might be better for it.